controller/work_item_children_blackbox_test.go

```diff
@@ -61,8 +62,14 @@ func (s *workItemChildSuite) SetupSuite() {
 	s.clean = cleaner.DeleteCreatedEntities(s.DB)
 
 	// Make sure the database is populated with the correct types (e.g. bug etc.)
+	ctx := migration.NewMigrationContext(context.Background())
```

Comment:
This has changed cause we handle these population in the test db suite. Check the other tests ;)